common/frame.go

```diff
@@ -1340,27 +1313,26 @@ func (f *Frame) SetContent(html string, opts goja.Value) {
 		forceCallable: true,
 		returnByValue: true,
 	}
+	rt := f.vu.Runtime()
 	if _, err := f.evaluate(f.ctx, utilityWorld, eopts, rt.ToValue(js), rt.ToValue(html)); err != nil {
-		k6common.Throw(rt, err)
+		k6Throw(f.ctx, "error setting content: %w", err)
```

Comment:
That is a nice one, maybe "wrap" with the details in other places (I see that you did it in some places :muscle: ) in this file? 

I mean to avoid this:
```go
k6Throw(f.ctx, "%s", err).
``` 

or maybe, as the minimal change, make it more flexible with:

```go
k6Throw(f.ctx, "%w", err).
``` 

What do you think?

Reply:
Heh I started doing this in some places, but gave up as it was a lot of work to come up with appropriate messages :sweat_smile: Plus I think @inancgumus and me disagree on the phrasing (he tends to use "unable to" and "cannot" whereas I think "error" is clearer), so I didn't want to add more discussion points.

Re: `%s` vs `%w`, sure, I'll change it, though there's no need to wrap these since `k6Throw` doesn't unwrap them and simply panics with the error, so `%s` would be fine as well.

Reply:
Did the minimal change in 934d913.